config/config.go

```diff
@@ -98,9 +98,8 @@ func (cfg *Config) Load(file string) error {
 			return err
 		}
 		var snippetdirs []string
-		cfg.General.SnippetFile = expandPath(cfg.General.SnippetFile)
```

Comment:
not directly clear to me why we need to remove the expand path from here

Reply:
This is where this issue is created. This line expand the path and this attribute is what gets written to file.

In this PR, my approach is to keep `cfg.General.SnippetFile` as is without expanding and expand whenever this file is to be checked/read/opened.

Reply:
Ah I thought so. Makes sense